data/crac-file/crac-api/src/main/java/com/farao_community/farao/data/crac_api/AbstractIdentifiable.java

```diff
@@ -42,4 +42,20 @@ public String toString() {
         return id;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
```

Comment:
Same remark for all similar methods in the different classes

Reply:
Literature has no definitive answer for this topic, but most people say that equals method should not throw exception. It says that in some cases where objects must absolutely not be null you can throw NullPointerException but that's not very clear. It seems that generally the common practice is to not throw exception, so until we encounter a situation where it will be really useful we let it like that.